api/src/test/java/org/openmrs/ui/framework/formatter/FormatterServiceTest.java

```diff
@@ -63,11 +62,10 @@ public void testMessage() throws Exception {
 		formatterService.setMessageSource(messageSource);
 		
 		String result = formatter.format(new Obs(), Locale.ENGLISH);
-		verify(messageSource).getMessage("testing.123.testing", null, Locale.ENGLISH);
```

Comment:
Are these really the same?

Reply:
I just noticed the old test passes when run individually. It only fails when running with the other test cases.

Reply:
Ofcourse you can always test and confirm your theory without first seeking permission from me. 😊

Reply:
I fixed the issue.